experimental/packages/api-logs/src/types/LogRecord.ts

```diff
@@ -14,46 +14,48 @@
  * limitations under the License.
  */
 
-import { Attributes } from '@opentelemetry/api';
+import type { Attributes, AttributeValue } from "@opentelemetry/api";
+
+export enum SeverityNumber {
+  UNSPECIFIED = 0,
+  TRACE = 1,
+  TRACE2 = 2,
+  TRACE3 = 3,
+  TRACE4 = 4,
+  DEBUG = 5,
+  DEBUG2 = 6,
+  DEBUG3 = 7,
+  DEBUG4 = 8,
+  INFO = 9,
+  INFO2 = 10,
+  INFO3 = 11,
+  INFO4 = 12,
+  WARN = 13,
+  WARN2 = 14,
+  WARN3 = 15,
+  WARN4 = 16,
+  ERROR = 17,
+  ERROR2 = 18,
+  ERROR3 = 19,
+  ERROR4 = 20,
+  FATAL = 21,
+  FATAL2 = 22,
+  FATAL3 = 23,
+  FATAL4 = 24,
+}
 
 export interface LogRecord {
```

Comment:
Why make this an interface that has to be implemented by a class? It seems to me like you are trying to mix SDK ideas into the API

Reply:
No, it's just a declaration, it's just an api，This is just a declaration. It only provides the api. It is the embodiment of the data structure. However, it should also provide some corresponding methods for data operation, such as `emit` and `setAttribute`.  It should be used like this: `logger.getLogRecord({body: "body"}).emit()` , it is not involved in the implementation and does not mix with the implementation, as mentioned above, this is the same as the implementation of `tracer.startSpan().end` and `meter.createCounter().add()`, But it is true that when implemented by the sdk they should all be implemented by classes, like: Span/CounterInstrument they are both classes